schemata/text.yaml

```diff
@@ -9,7 +9,15 @@ properties:
   id:
     "$ref": "base.json#/definitions/uuid"
   type:
-    "$ref": "contentblock.json#/definitions/type"
+    description: Block type
+    example: "text"
+    type: string
+    enum:
+      - text
+      - code
+      - quote
+      - hr
+      - unknown
```

Comment:
@bergie is it guaranteed that `unknown` blocks have all the properties of a `text` block? If yes, why isn't it  just called `text`?